dpop/DPoP-client-app/src/main/java/org/wso2/dpop/client/GeneratePublicKeyPair.java

```diff
@@ -0,0 +1,62 @@
+package org.wso2.dpop.client;
```

Comment:
add license txt. fix in other places too

Reply:
Done